reference/commands/install.rst

```diff
@@ -1,2 +1,273 @@
+- Download precompiled binaries, or build binaries from sources in the local cache, in the right order for the dependency graph.
+- Create the necessary files as requested by the "generators", so build systems and other tools can locate the locally installed dependencies
+- Optionally, execute desired ``deployers``.
+
+
+Conanfile path or --requires
+----------------------------
+
+The ``conan install`` command can use 2 different origins for information. The first one is using a local ``conanfile.py`` 
+or ``conanfile.txt``, containing definitions of dependencies, generators to be used.
+
+.. code-block:: bash
+
+    $ conan install .  # there is a conanfile.txt or a conanfile.py in the cwd
+    $ conan install conanfile.py  # also works, direct reference file
+    $ conan install myconan.txt  # explicit custom name
+    $ conan install myfolder  # there is a conanfile in "myfolder" folder
+
+
+Even if it is possible to use a custom name, in the general case, it is recommended to use the default ``conanfile.py`` 
+name, located in the repository root, so users can do a straightforward ``git clone ... `` + ``conan install .``
+    
+
+The other possibility is to not have a ``conanfile`` at all, and define the requirements to be installed directly in the
+command line:
+
+.. code-block:: bash
+
+    # Install the zlib/1.2.13 library
+    $ conan install --requires=zlib/1.2.13
+    # Install the zlib/1.2.13 and bzip2/1.0.8 libraries
+    $ conan install --requires=zlib/1.2.13 --requires=bzip2/1.0.8
+    # Install the cmake/3.23.5 and ninja/1.11.0 tools
+    $ conan install --tool-requires=cmake/3.23.5 --tool-requires=ninja/1.11.0
+    # Install the zlib/1.2.13 library and ninja/1.11.0 tool
+    $ conan install --requires=zlib/1.2.13 --tool-requires=ninja/1.11.0
+
+
+In the general case, it is recommended to use a ``conanfile`` instead of defining things in the command line.
+
+
+Profiles, Settings, Options, Conf
+---------------------------------
+
+There are several arguments that are used to define the effective profiles that will be used, both for "build"
+and "host" contexts.
+
+By default the arguments refer to the "host" context, so ``--settings:host, -s:h`` is totally equivalent to
+``--settings, -s``. Also, by default, the ``conan install`` command will use the ``default`` profile both for the
+"build" and "host" context. That means that if a profile with the "default" name has not been created, it will error.
+
+Multiple definitions of profiles can be passed as arguments, and they will compound from left to right (right has the
+highest priority)
+
+.. code-block:: bash
+
+    # The values of myprofile 3 will have higher priority
+    $ conan install . -pr=myprofile1 -pr=myprofile2 -pr=myprofile3
+
+If values of ``settings``, ``options`` and ``conf`` are provided in the command line, they create a profile that
+is composed with the other provided ``-pr`` (or the "default" one if not specified) profiles, with higher priority,
```

Comment:
```suggestion
is composed with the other provided ``-pr`` profiles (or the "default" one if not specified), which has the highest priority,
```

Reply:
Maybe this one is not clear, it is composed, but not clear if the which applies to the "default" profile (it doesn't, it applies to the settings and options one)